homeassistant/components/zha/repairs.py

```diff
@@ -0,0 +1,126 @@
+"""ZHA repairs for common environmental and device problems."""
+from __future__ import annotations
+
+import enum
+import logging
+
+from universal_silabs_flasher.const import ApplicationType
+from universal_silabs_flasher.flasher import Flasher
+
+from homeassistant.components.homeassistant_sky_connect import (
+    hardware as skyconnect_hardware,
+)
+from homeassistant.components.homeassistant_yellow import (
+    RADIO_DEVICE as YELLOW_RADIO_DEVICE,
+    hardware as yellow_hardware,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import issue_registry as ir
+
+from .core.const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class AlreadyRunningEZSP(Exception):
+    """The device is already running EZSP firmware."""
+
+
+class HardwareType(enum.StrEnum):
+    """Detected Zigbee hardware type."""
+
+    SKYCONNECT = "skyconnect"
+    YELLOW = "yellow"
+    OTHER = "other"
+
+
+DISABLE_MULTIPAN_URL = {
+    HardwareType.YELLOW: (
+        "https://yellow.home-assistant.io/guides/disable-multiprotocol/#flash-the-silicon-labs-radio-firmware"
+    ),
+    HardwareType.SKYCONNECT: (
+        "https://skyconnect.home-assistant.io/procedures/disable-multiprotocol/#step-flash-the-silicon-labs-radio-firmware"
```

Comment:
```suggestion
        "https://skyconnect.home-assistant.io/procedures/multiprotocol-firmware-detected/"
```
Since this message is a bit more complex now, but the Learn more button can only link to one page, I think I will add an extra page for it in the docs. Same content as used for Yellow.